### PR TITLE
Updating readme: setting bootstrap script architecture to 64 bits

### PR DESCRIPTION
**Pull request by carolahp**
The current script displays '32 bits', executing this script fails while trying to download the VM

### CODE REVIEW

**Comment by MarcusDenker** [comment]
failing tests on win seem to be unrelated:

```
windows-64 / Tests-windows-64 / testArchiveWithThousandFilesShouldWork – Windows64.Compression.Tests.Archives.ZipArchiveTest
windows-64 / Tests-windows-64 / testInstallFromFile – Windows64.System.Installers.Tests.MCMczInstallerTest
windows-64 / Tests-windows-64 / testRespond – Windows64.Zinc.Tests.ZnServerTest
```

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ To bootstrap a new Pharo image you need the latest stable version of Pharo. For
 The bootstrapping can be done on a properly-named branch using the following script:
 
 ```
-BUILD_NUMBER=42 BOOTSTRAP_ARCH=32 bash ./bootstrap/scripts/bootstrap.sh
+BUILD_NUMBER=42 BOOTSTRAP_ARCH=64 bash ./bootstrap/scripts/bootstrap.sh
 ```
 
 This will generate and archive images at various stages of the bootstrap process up to the full image in Pharo8.0-32bit-hhhhhhh.zip where hhhhhhh is the hash of the current checkout.
```
Comment (astares) [inline README.md]: Is this then a Pharo8.0-**64**bit-hhhhhhh.zip or still Pharo8.0-**32**bit-hhhhhhh.zip

Comment (tesonep) [reply README.md]: Good catch, I fixed it.

Comment (astares) [reply README.md]: Thanks!

**Pull request merged**